docs/user-guides/c-types-of-layers/k.trip.md

```diff
@@ -0,0 +1,59 @@
+# Trip layer
+
+![Trip layer](https://d1a3f4spazzrp4.cloudfront.net/kepler.gl/documentation/k-trip.gif 'Grid layer')
+Trip layer can display animated path.
+
+### How to use trip layer to animate path
+
+**Data format**
+Currently trip layer support a special `geoJSON` format where the coordinate `linestring` has a 4th element denoting timestamp.
+
+In order to animate the path, the `geoJSON` data needs to contain `LineString` in its feature geometry, and the coordinates in the `LineString` need to have 4 elements in the format of  `[longitude, latitude, altitude, timestamp]` with the last element being a valid unix timestamp.
```

Comment:
unix timestamp in miliseconds? or seconds? You can link to
https://www.unixtimestamp.com/index.php